internal/bgp/frr/frr_test.go

```diff
@@ -107,7 +137,7 @@ func TestSingleSessionClose(t *testing.T) {
 	sessionManager := NewSessionManager(l)
 	defer close(sessionManager.reloadConfig)
 
-	session, err := sessionManager.NewSession(l, "10.2.2.254:179", net.ParseIP("10.1.1.254"), 100, net.ParseIP("10.1.1.254"), 200, time.Second, time.Second, "password", "hostname", "")
+	session, err := sessionManager.NewSession(l, "10.2.2.254:179", net.ParseIP("10.1.1.254"), 100, net.ParseIP("10.1.1.254"), 200, time.Second, time.Second, "password", "hostname", "", true)
```

Comment:
Can we have at least one singlehop session in tests?

Reply:
Never mind, I saw you added new tests above for that.

Reply:
Added here and in `TestParse`